config/default.js

```diff
@@ -15,4 +15,5 @@ module.exports = {
     },
     hostname: 'http://data.istex.fr/',
     uploadDir: 'upload',
+    istexApiUrl: 'https://api.istex.fr/document',
```

Comment:
Please, can you set this config like: `https://api.istex.fr` with no slash at the end?